docs/crio.conf.5.md

```diff
@@ -206,6 +206,9 @@ the container runtime configuration.
 **read_only**=false
   If set to true, all containers will run in read-only mode.
 
+**enable_custom_shm_size**=false
+  If set to true, enable users to set custom shm size instead of default value 64M. The shmsize can be set through K8S annotation with key "io.kubernetes.cri-o.ShmSize", and the value representing the size in bytes.
```

Comment:
```suggestion
  If set to true, enable users to set a custom shm size instead of using the default value of 64M. The shm size can be set through K8S annotation with the key "io.kubernetes.cri-o.ShmSize", and the value representing the size in bytes.
```